scripts/ci/in_container/krb5/krb5.conf

```diff
@@ -17,8 +17,8 @@
 # under the License.
 
 [logging]
-default = FILE:/var/log/krb5libs.log
-kdc = FILE:/var/log/krb5kdc.log
+default = FILE:/var/log/kerberos.log
```

Comment:
Why do we want to have these logs in one file?

Reply:
Mistake ! Thanks. I will fix it.